.github/workflows/verify.yml

```diff
@@ -10,9 +10,16 @@ jobs:
       - uses: ./.github/actions/yarn_install
       - run: yarn lint
       - run: yarn build
-      - run: yarn test
+      - name: Run unit tests and publish to Code Climate
+        env:
+          CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+        run: |
+          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
```

Comment:
in sense-client we do use localy stored version of cc-test-reporter to remove risk on foreign injection
do we also want that here?

Reply:
Sure, can fix that.